pkgs/development/python-modules/qscintilla-qt5/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, qscintillaCpp
+, lndir
+, sip
+, python
+, pyqt5 }:
+
+buildPythonPackage rec {
+  pname = "qscintilla";
+  version = qscintillaCpp.version;
+  src = qscintillaCpp.src;
+  format = "other";
+
+  buildInputs = [ lndir qscintillaCpp sip ];
```

Comment:
Shouldn't `sip` be propagated?

Reply:
sip is actually a binding generator tool. Is is not required at runtime. It generates artefacts. I am just not sure if it should go into `nativeBuildInputs` of if `buildInputs` is fine.

Reply:
I opted for the `nativeBuildInputs` approach

Reply:
I'm still not sure about that since I don't know anything about cross-compiling Python packages.